### PR TITLE
Yealink v81 w52 56 new firmware

### PR DESCRIPTION
**Pull request by fouille**

### CODE REVIEW

File: plugins/xivo-yealink/v81/pkgs/pkgs.db [hunk 1]
```diff
@@ -274,10 +274,10 @@ size: 18948304
 sha1sum: 7bd04e6f92e94ff819f4694f502e63e6bc1079b1
 
 [file_W52P_W56P-base-fw]
-filename: Base-W52P-W56P-25.81.0.10.rom
-url: http://download.support.yealink.com/download?path=ZIjHOJbWuW/DFrGTLnGypsN1StLgAwqIbXplusSymboleOhpPym/bPNc3lNFOcMcJBzGqUr/8syZXHHLlYcAGh9Mt9WObDk1QwXUClUp6U3PnQtJon0I5ET5BfqI8dRS8liplusSymbolyVEg29utctf2JmplusSymbol8=
+filename: Base-W52P-W56P-25.81.0.60.rom
+url: https://support.yealink.com/forward2download?path=ZIjHOJbWuW/DFrGTLnGypgRplusSymbolYFVgEdMpoBjb4z7QlPnbSdqizjP/cuPAJea/izs000vYrgAxHUMMf6ZvgrlHOJZGVrwbsS7UkuT40UqtNbSgyWSVRBtG2EHFWHFj1HSPKftWtvfBzd4=
 size: 6763776
```
Comment (bloom1) [inline plugins/xivo-yealink/v81/pkgs/pkgs.db:279]: same size?

Comment (fouille) [reply plugins/xivo-yealink/v81/pkgs/pkgs.db:279]: yes  : 
``` 
-rw-r--r-- 1 root root  6763776 Mar 12 17:26 w52.rom
```

Comment (fouille) [reply plugins/xivo-yealink/v81/pkgs/pkgs.db:279]: tested with Zuul link, work's perfectly

Comment (bloom1) [reply plugins/xivo-yealink/v81/pkgs/pkgs.db:279]: > yes :
> 
> ```
> -rw-r--r-- 1 root root  6763776 Mar 12 17:26 w52.rom
> ```

Surprising! LGTM then :smile:

**Review by bloom1 — APPROVED** [review]

**Pull request merged**